front-end-web/src/Components/Pages/AuthorizationPage.tsx

```diff
@@ -9,11 +11,24 @@ const AuthorizationPage: React.FunctionComponent<Props> = ({
   handlers,
   classes
 }) => {
+  const useStyles = makeStyles(theme => ({
+    button: {
+      margin: "8px"
+    },
+    input: {
+      display: "none"
+    }
+  }));
+
+  const classes2 = useStyles();
   return (
     <Fragment>
-      {
-        //TODO
-      }
+      <Typography>
+        Press this button to log in to the Authorization page
+      </Typography>
+      <Button variant="outlined" color="primary" className={classes.button}>
```

Comment:
4 things, all minor, but if you could get these updated that would be greatly appreciated!
1.  Let's switch variant from `outlined` to `contained`. This will make it stand out more.
2.  Related to number one, let's add the `size="large"` tag as well. This should make it easier to read on a projector demo.
3.  You've declared `classes2` above, so let's switch this to `classes2.button` so the style shows up.
4.  Can you add `onClick={handlers.handleAuth}`? This is the new code that will make the back end work.

Reply:
I've made the changes, but I have 2 errors:
1. When I changed `classes2` to `classes2.button` I am getting an error under `classes2.`
2. I am getting an error where I put the `onClick={handlers.handleAuth}` on the `handlers.` part